### PR TITLE
changed command for redhat family to modprobe to properly evaluate test

### PR DESCRIPTION
**Pull request by crashdummymch**
modinfo does not produce the correct output for the test and should use modprobe.  Test currently always fails when using modinfo due to output not matching.

### CODE REVIEW

**Comment by chris-rock** [comment]
@crashdummymch Thank you very much, once you've rebased, this should turn green.

**Comment by chris-rock** [comment]
Is this fixing #4?

**Comment by crashdummymch** [comment]
this fixes #4

File: libraries/linux_module.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def version
 
   def command
     modinfo_cmd = if inspec.os.redhat? || inspec.os.name == 'fedora'
-                    "/sbin/modinfo -n -v #{@module}"
+                    "/sbin/modprobe -n -v #{@module} | awk '{$1=$1;print}'"
                   else
                     "modinfo -n -v #{@module}"
```
[the remark this answers is not in the document]
Comment (crashdummymch) [reply libraries/linux_module.rb]: removed condition logic so that all linux systems utilize modprobe for detecting module status

**Pull request merged**